internal/payloads/permissions_payload.go

```diff
@@ -0,0 +1,28 @@
+package payloads
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+)
+
+type PermissionsResponse struct {
+	Valid           bool
+	MissingPolicies []string
+}
+
+func (s *PermissionsResponse) Bind(_ *http.Request) error {
+	return nil
+}
+
+func (s *PermissionsResponse) Render(_ http.ResponseWriter, _ *http.Request) error {
+	return nil
+}
```

Comment:
Response needs Render. Reply needs Bind. I think we got that incorrectly in other parts of the app too, no big deal.

Reply:
Okay, let's get rid of Bind.